cogs/tasks.py

```diff
@@ -52,6 +55,21 @@ async def distribute_monthly_quiz_rewards(self) -> None:
                 if rank in roles.keys() and roles[rank] is not None:
                     await user.add_roles(roles[rank])
 
+    @tasks.loop(minutes=30)
+    async def drop_card(self) -> None:
+        cards = iufi.CardPool.roll(amount=1)
+        channel = self.bot.get_channel(random.choice(func.settings.GAME_CHANNEL_IDS))
+        if channel:
+            view = DropView(cards)
+            covered_card: iufi.TempCard = iufi.TempCard(f"cover/level{random.randint(1, 3)}.webp")
+            image_bytes, image_format = await asyncio.to_thread(covered_card.image_bytes), covered_card.format
+            view.message = await channel.send(
+                content=f"**Hurry up! This claim ends in: <t:{round(time.time()) + 70}:R>**",
+                embed=view.build_embed(),
+                file=discord.File(image_bytes, filename=f'image.{image_format}'),
+                view=view
+            )
```

Comment:
can't we add some random wait/sleep timer here? otherwise people might just figure out the timer and will just wait on all channels when clock hit the exact time for drop. it wouldn't truly be random.